src/components/home/Home.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <section>
+    <h2>Welcome!</h2>
+    <RouterLink to="/news-search"></RouterLink>
```

Comment:
```suggestion
    <RouterLink to="/arts">Click here to start!</RouterLink>
```
Currently, there is no "news-search" route in router.js, so this link will not display anything.